init.lua

```diff
@@ -236,6 +241,34 @@ minetest.register_chatcommand("warp", {
 	end
 })
 
+local function prepare_dropdown(x,y,w,h,curr_dest)
+	local dd = string.format("dropdown[%f,%f;%f,%f;ddwarp;", x, y, w, h)
+	local sel = 0
+	for idx, warp in ipairs(warps) do
+		local warpname = warp.name
+		dd = dd .. warpname .. ","
```

Comment:
```suggestion
		dd = dd .. minetest.formspec_escape(warpname) .. ","
```
This avoids formspec errors if the warp name contains any of `,;[]`.